plugin/kubernetai/kubernetai.go

```diff
@@ -83,14 +88,32 @@ func (k8i Kubernetai) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dns
 
 // AutoPath routes AutoPath requests to the authoritative kubernetes.
 func (k8i Kubernetai) AutoPath(state request.Request) []string {
+	var searchPath []string
 	for _, k := range k8i.Kubernetes {
-		zone := plugin.Zones(k.Zones).Matches(state.Name())
-		if zone == "" {
-			continue
+		zones := make([]string, 0, len(k.Zones)*2)
+		zones = append(zones, k.Zones...)
+		for _, z := range k.Zones {
+			if !strings.HasPrefix(z, "svc.") {
+				zones = append(zones, "svc."+z)
+			}
+		}
+		zone := plugin.Zones(zones).Matches(state.Name())
+		if zone != "" {
+			searchPath = append([]string{zone}, searchPath...)
+			ip := state.IP()
+			pods := k.APIConn.PodIndex(ip)
+			var pod *api.Pod = nil
+			if len(pods) != 0 {
+				pod := ps[0]
```

Comment:
~~~
plugin/kubernetai/kubernetai.go:107:12: undefined: ps
Makefile:11: recipe for target 'coredns' failed
~~~